web/src/add_streams_pill.ts

```diff
@@ -0,0 +1,89 @@
+import * as input_pill from "./input_pill";
+import {set_up} from "./pill_typeahead";
```

Comment:
Same dependency problem is faced by #24580. We probably have to wait for `pill_typeahead` ts conversion.

related CZO discussion [here](https://chat.zulip.org/#narrow/stream/6-frontend/topic/input_pill.20typescript/near/1670357).

Reply:
It's probably reasonable to wait for that module to migrate, since it's being actively worked on.